CHANGES

```diff
@@ -1,5 +1,19 @@
 Version 2.0
 ===========
+Backwards Incompatible Changes
+------------------------------
+* Any installed ZenPacks using older versions of zenpacklib.py will continue to function unchanged.
+* Using version 2.0 is slightly different.  The __init__.py file import statements should now contain the following:
+
+.. code-block:: python
+
+	from ZenPacks.zenoss.ZenPackLib import zenpacklib
+	CFG = zenpacklib.load_yaml()
+	schema = CFG.zenpack_module.schema
+
+* zProperties will not be updated automatically on existing device classes.  These should be handled on a case basis by using migrate scripts
+* Further documentation of these and other changes can be found at `<http://zenpacklib.zenoss.com/en/latest/>`_.
```

Comment:
I'm not sure that stating that you can read more about the changes by reading all of the documentation is useful.